docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+---
+title: File Access Authorization
+parent: Deployment
+nav_order: 4
+---
+
+# (BETA) File Access Authorization
+
+> **IMPORTANT:** This feature is currently in beta. Configuration and log formats are subject to change.
```

Comment:
If there are a couple of things you'd like to call out especially for issues that are to be fixed, you could group under a ## Known Limitations header. This might be a bit excessive here, but worth considering. Providing even more transparency with a linked GH issue means that users can also follow the issue if they wish. Just a thought!

Reply:
I like the idea of a "Known Limitations" section, but I wasn't able to make it look right to me. Things are currently ordered in a way that introduces concepts and groups the limitations next to each other. Breaking that out would remove that grouping and potentially make it more difficult to discover pertinent information.

Re: calling out the delay - I expanded a bit up top about the update interval key existing and added and link to the section with more details.